automatic/ruby/README.md

```diff
@@ -2,16 +2,8 @@
 
 Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
 
-This package provides a self-contained [Windows-based installer](https://rubyinstaller.org) that includes the Ruby language, an execution environment, important documentation, and more.
-
-## Package Parameters
-
-- `/InstallDir` - Ruby installation directory, by default `c:\tools\RubyXY` where XY are major and minor version parts.
-- `/NoPath`     - Do not add ruby bin folder to machine PATH.
-
-Example: `choco install ruby --params "/NoPath /InstallDir:C:\your\install\path"`
-
 ## Notes
 
-- To install ruby development kit ruby installer provides `ridk` command. It provides easy way to install msys2 via `ridk install 1`, however, the installation is interactive. To accomplish unattended install, use [msys2](https://chocolatey.org/packages/msys2) package.
+- This package depends on the the latest major version of the ruby.install package.
+- This package do not support package parameters, please install the package [ruby.install](https://chocolatey.org/packages/ruby.install) directly when using package parameters.
```

Comment:
Should be *does not*. Same thing in `ruby.nuspec`.

Reply:
Fixed.